src/chapters/methods/exercises.rst

```diff
@@ -0,0 +1,188 @@
+Exercises: Methods
+=====================
+
+To become good at solving problems with code you need to be able to break large
+problems into small ones. Usually, these smaller problems will take the form of
+methods that are used to solve the larger problem. Therefore, to become good
+at solving problems you need to become good at writing methods. And to master
+methods, you need to write a *lot* of them.
+
+These exercises ask you to write lots of methods, starting with 
+small one and end with a larger, more complicated one.
+
+At the end, you will be able to create strings of shapes, like this nifty
+diamond:
+
+::
+
+       #
+      ###
+     #####
+    #######
+   #########
+   #########
+    #######
+     #####
+      ###
+       #
+
+.. admonition:: Note
+
+   As you build your methods, remember to keep the output in mind.  
+   Do you want a value returned or are do you want the method to print directly to the console when invoked?
+   How many things are being repeated?
+
+There is no starter code for these exercises, so create a new C# project
+at `repl.it <https://repl.it/login>`__ to get started.
+
+.. admonition:: Tip
+
+   As you create these methods, keep the shape of your output in mind.  
+   Remember the differences between ``Console.WriteLine`` and ``Console.Write`` .  
+
+
+Rectangles
+----------
+
+#. Write a method ``MakeLine(size)`` that returns a line with exactly ``size``
+   hashes.
+
+   .. sourcecode:: csharp
+
+      MakeLine(5);
+
+   **Console Output**
+
+   ::
+
+      #####
+
+#. Write a method called ``MakeSquare(size)`` that returns a ``size`` by
+   ``size`` iteration of hashes. 
+
+   .. sourcecode:: csharp
+
+      MakeSquare(5);
+
+   **Console Output**
+
+   ::
+
+      #####
+      #####
+      #####
+      #####
+      #####
+
+Asthetically it's not a "perfect" square, but a single argument led to its creation.
+
+#. Write a method ``MakeRectangle(width, height)`` that returns a
```

Comment:
this can be modified to be called 3. (right now it renders as 1. because of the text line breaking this up into 2 sections)